src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
I think we would want to add the border when we call  `query_overlapping_fieldids(ramin, ramax, decmin, decmax)` in `infer_nersc`. The `find(entry_in_range, catalog)` should only get sources in the target sky patch.

BTW, this code hasn't changed in this PR - this was just a variable rename for consistency.